### ISSUE TITLE
Component: slider - cannot disable on first render

### ISSUE THREAD

**Issue by andrkole** [primefaces/primeng#12587]
### Describe the bug

p-slider doesn't show as disabled when the page displays, only after the page is rendered if the value of property tied to [disabled] is updated it will show as disabled (greyed out).

Even if I set the [disabled] property to true without any dynamic value ([disabled="true"), it still shows the slider as active.

It seems that the p-slider component actually sees the disabled property is set to true, but it doesn't set the child's classes to disabled:
<img width="613" alt="image" src="https://user-images.githubusercontent.com/44214882/215813335-1755a912-87e9-4884-85a5-e564e5aec5e1.png">
<img width="742" alt="image" src="https://user-images.githubusercontent.com/44214882/215813877-8e8feeac-b006-4abc-aaf3-b84b2256e375.png">

### Environment

Angular: 15.1.2
PrimeNG:  15.2.0

### Angular version

15.1.2

### PrimeNG version

15.2.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.13.0

### Steps to reproduce the behavior

1. Add a <p-slider> component and set it's property [disabled] to true
2. Start the app
3. Slider shows as enabled on the page

### Expected behavior

Slider should be disabled upon the first render of the page

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap?
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
